src/contracts/pods/EigenPodManager.sol

```diff
@@ -314,12 +330,38 @@ contract EigenPodManager is
     }
 
     /// @notice Returns the Beacon block root at `timestamp`. Reverts if the Beacon block root at `timestamp` has not yet been finalized.
-    function getBlockRootAtTimestamp(uint64 timestamp) external view returns (bytes32) {
+    function getBlockRootAtTimestamp(uint64 timestamp) public view returns (bytes32) {
         bytes32 stateRoot = beaconChainOracle.timestampToBlockRoot(timestamp);
         require(
             stateRoot != bytes32(0),
             "EigenPodManager.getBlockRootAtTimestamp: state root at timestamp not yet finalized"
         );
         return stateRoot;
     }
+
+    //make a request for a proof to the succinct gateway
+    function requestProofViaSuccinctGateway(
+        bytes32 FUNCTION_ID,
+        uint64 startSlot,
+        uint64 endSlot,
+        address podAddress,
+        bytes memory callbackData,
+        uint32 callbackGasLimit
+    ) external payable {
```

Comment:
hmm, probably want this access-restricted to EigenPods?

Reply:
yeah good call